### PR TITLE
[FEATURE] csp: Add server documentation

### PR DESCRIPTION
**Pull request by tobiasso85**
Add documentation for CSP Reporting

### CODE REVIEW

File: docs/pages/Server.md [hunk 1]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
...
 
-**Hint:** If Chrome unintentionally redirects a HTTP-URL to HTTPS, you need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. localhost) and pressing "delete".
+**Hint:** If Chrome unintentionally redirects an HTTP-URL to HTTPS, you need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. localhost) and pressing "delete".
+
+## CSP Reporting
+The support of a content security policy ([CSP](https://www.w3.org/TR/CSP/)) is active by default.
+The header `content-security-policy` can be set by adding URL parameter `sap-ui-xx-csp-policy` to the request with the policy name as value.
+To set the policy to report-only append `:report-only` or `:ro` to the policy name.
+E.g. `/index.html?sap-ui-xx-csp-policy=sap-target-level-1:report-only`
+
```
Comment (RandomByte) [inline docs/pages/Server.md:22]: Please use just one new line between paragraphs like above

File: docs/pages/Server.md [hunk 2]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
...
+
+### SAP target CSP
+The CSP policies can be modified using parameter `sendSAPTargetCSP`.
+With `sendSAPTargetCSP` the policies `sap-target-level-1` and `sap-target-level-2` policies are activated and send as report-only.
```
Comment (RandomByte) [inline docs/pages/Server.md:26]:
```suggestion
With `sendSAPTargetCSP` set to `true` the policies `sap-target-level-1` and `sap-target-level-2` policies are activated and send as report-only.
```

File: docs/pages/Server.md [hunk 3]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
...
+
+## CSP Reporting
+The support of a content security policy ([CSP](https://www.w3.org/TR/CSP/)) is active by default.
+The header `content-security-policy` can be set by adding URL parameter `sap-ui-xx-csp-policy` to the request with the policy name as value.
+To set the policy to report-only append `:report-only` or `:ro` to the policy name.
+E.g. `/index.html?sap-ui-xx-csp-policy=sap-target-level-1:report-only`
+
+
+### SAP target CSP
+The CSP policies can be modified using parameter `sendSAPTargetCSP`.
```
Comment (RandomByte) [inline docs/pages/Server.md:25]:
```suggestion
The default CSP policies can be modified using parameter `sendSAPTargetCSP` (`--sap-csp-policies` when using the CLI).
```

File: docs/pages/Server.md [hunk 4]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
...
+With `sendSAPTargetCSP` the policies `sap-target-level-1` and `sap-target-level-2` policies are activated and send as report-only.
+
+
+### Serve CSP Reports
+Serving of CSP reports can be activated with parameter `serveCSPReports`.
```
Comment (RandomByte) [inline docs/pages/Server.md:30]:
```suggestion
Serving of CSP reports can be activated with parameter `serveCSPReports` (`--serve-csp-reports` when using the CLI).
```

File: docs/pages/Server.md [hunk 5]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
...
+With `serveCSPReports` the csp reports are collected and can be downloaded using the request path `/.ui5/csp/csp-reports.json`.
```
Comment (RandomByte) [inline docs/pages/Server.md:31]:
```suggestion
With `serveCSPReports` set to `true`, the CSP reports are collected and can be downloaded from the server path `/.ui5/csp/csp-reports.json`.
```

File: docs/pages/Server.md [hunk 6]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
 ## Certificates for HTTPS or HTTP/2
 When using the UI5 Server with the UI5 CLI, SSL certificates for HTTPS and HTTP/2 configurations can automatically be generated.
 
-**Hint:** If Chrome unintentionally redirects a HTTP-URL to HTTPS, you need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. localhost) and pressing "delete".
+**Hint:** If Chrome unintentionally redirects an HTTP-URL to HTTPS, you need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. localhost) and pressing "delete".
+
+## CSP Reporting
+The support of a content security policy ([CSP](https://www.w3.org/TR/CSP/)) is active by default.
+The header `content-security-policy` can be set by adding URL parameter `sap-ui-xx-csp-policy` to the request with the policy name as value.
+To set the policy to report-only append `:report-only` or `:ro` to the policy name.
```
Comment (RandomByte) [inline docs/pages/Server.md:20]:
```suggestion
To set the policy to report-only, append `:report-only` or `:ro` to the policy name.
```

File: docs/pages/Server.md [hunk 7]
```diff
@@ -12,4 +12,20 @@ The development server has already a set of middleware which supports the develo
 ## Certificates for HTTPS or HTTP/2
 When using the UI5 Server with the UI5 CLI, SSL certificates for HTTPS and HTTP/2 configurations can automatically be generated.
 
-**Hint:** If Chrome unintentionally redirects a HTTP-URL to HTTPS, you need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. localhost) and pressing "delete".
+**Hint:** If Chrome unintentionally redirects an HTTP-URL to HTTPS, you need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://net-internals/#hsts) by entering the domain name (e.g. localhost) and pressing "delete".
+
+## CSP Reporting
+The support of a content security policy ([CSP](https://www.w3.org/TR/CSP/)) is active by default.
```
[the remark this answers is not in the document]
Comment (tobiasso85) [reply docs/pages/Server.md:18]: done

File: docs/pages/Server.md (hunk 1 above)
Comment (tobiasso85) [reply docs/pages/Server.md:22]: done

File: docs/pages/Server.md (hunk 2 above)
Comment (tobiasso85) [reply docs/pages/Server.md:26]: done

File: docs/pages/Server.md (hunk 3 above)
Comment (tobiasso85) [reply docs/pages/Server.md:25]: done

File: docs/pages/Server.md (hunk 4 above)
Comment (tobiasso85) [reply docs/pages/Server.md:30]: done

File: docs/pages/Server.md (hunk 5 above)
Comment (tobiasso85) [reply docs/pages/Server.md:31]: done

File: docs/pages/Server.md (hunk 6 above)
Comment (tobiasso85) [reply docs/pages/Server.md:20]: done

**Pull request merged**